### DISCUSSION TITLE
Is there any Simple Analytics Plugins available?

### DISCUSSION THREAD

**Discussion by SanityZeroPercent** [facebook/docusaurus#8121] [Q&A] [answered]
Hey folks,

I was wondering to look after something than usual google analytics, and i already have Simple Analytics.
The questions is, is there any known available or community-run-plugins Simple Analytics?

Thanks.

**Comment by slorber** [comment, marked as answer]
I am not aware of such community plugin sorry. Only npm or Google can answer

**Comment by FMCorz** [comment]
I could not find any, and wrote one: https://github.com/branchup/docusaurus-plugin-simple-analytics
